aidlc-docs/inception/requirements/requirements.md

```
@@ -653,27 +672,34 @@ MVP でやらないこと:
 
 ### 6.2 2エージェント協調構成（Dual-Agent）
 
-> **バージョン注記**: 本図は v1.0.0（書類審査・M2 MVP）時点の2エージェント構成を示す。
+> **バージョン注記（v1.2.0）**: 本図は v1.0.0（予選 M2 MVP）時点の2エージェント構成を示す。  
+> AgentCore は廃止し、**converse API + Tool Use** による直接実装に変更した。  
 > v1.1.0 では TaskOrganizerAgent（エージェント①b）が①と②の間に追加される（§FR-01b 参照）。
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)